test/functional/wallet_sendall.py

```diff
@@ -379,6 +379,20 @@ def sendall_with_maxconf(self):
         assert_equal(len(self.wallet.listunspent()), 1)
         assert_equal(self.wallet.listunspent()[0]['confirmations'], 6)
 
+    @cleanup
+    def sendall_anti_fee_sniping(self):
+        self.log.info("Testing sendall does anti-fee-sniping when locktime is not specified")
+        self.add_utxos([10,11])
+        tx_from_wallet = self.test_sendall_success(sendall_args = [self.remainder_target])
+
+        assert_greater_than(tx_from_wallet["decoded"]["locktime"], tx_from_wallet["blockheight"] - 100)
+
+        self.add_utxos([10,11])
+        txid = self.wallet.sendall(recipients=[self.remainder_target], options={"locktime":0})["txid"]
+        tx_from_wallet = self.wallet.decoderawtransaction(self.wallet.getrawtransaction(txid))
```

Comment:
In 5101f054ce435ac653a3f5e780fca066e2f25088 "test: test sendall and send do anti-fee-sniping"

nit: `getrawtransaction` has a `verbosity` option that will do the decoding, so no need to call `decoderawtransaction` here.

Reply:
Done